tests/python/gaia-ui-tests/gaiatest/tests/functional/lockscreen/test_lockscreen_unlock_to_camera.py

```diff
@@ -25,6 +26,7 @@ def test_unlock_to_camera(self):
         lock_screen = LockScreen(self.marionette)
         lock_screen.switch_to_frame()
         camera = lock_screen.unlock_to_camera()
+        Wait(self.marionette).until(lambda m: self.device.is_locked == False)
```

Comment:
`unlock_to_passcode_pad()` sounds like a way to go, to me. 

For `test_lockscreen_unlock_to_emergency_call_screen.py`, I think we can wrap it the same way as `unlock_to_passcode_pad()`:
``` python
def unlock_to_emergency_call(self):
    passcode_pad = self.unlock_to_passcode_pad()
    return passcode_pad.tap_emergency_call()
```

@npark-mozilla What do you think?